### ISSUE TITLE
Dockerfile for the codenvy/mysql image?

### ISSUE THREAD

**Issue by kaloyan-raev** [eclipse-che/che#2991]
The Java-MySQL stack uses the codenvy/mysql docker image. I searched in the [codenvy/dockerfile](https://github.com/codenvy/dockerfiles) repository, but could not find the Dockerfile.

Could you publish the Dockerfile of this image? I want to study it, so I can better understand how to create a PHP-MySQL stack.

**Comment by TylerJewell** [comment]
I couldn't find it either.  It should be in the `github.com/eclipse/che-dockerfiles` repository and I do not see anything there.  I'm going to assign to @ddementieva and @eivantsov - there is probably something that just needs to be added to the repo.

**Comment by ghost** [comment]
For some reason an official mysql image does not work for me. So, I have to add my own CMD. It can be an entrypoint as well though.

```
FROM mysql
EXPOSE 3306
CMD mysql -uroot -e "CREATE USER '$MYSQL_USER'@'%' IDENTIFIED BY '"$MYSQL_PASSWORD"'" && \
    mysql -uroot -e "GRANT ALL PRIVILEGES ON *.* TO '$MYSQL_USER'@'%' IDENTIFIED BY '"$MYSQL_PASSWORD"'; FLUSH PRIVILEGES;" && \
    mysql -uroot -e "CREATE DATABASE $MYSQL_DATABASE;" && \
    tail -f /dev/null

```

**Comment by kaloyan-raev** [comment]
@eivantsov Thanks, this was helpful!

I investigated the problem with the official mysql image and pushed a fix with PR #3049.
